RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: minor
+
+This release blocks installation of Hypothesis on Python 3.3, which
+:PEP:`reached its end of life date on 2017-09-29 <398>`.
+
+This should not be of interest to anyone but downstream maintainers -
+if you are affected, migrate to a secure version of Python as soon as
+possible or at least seek commercial support.
```

Comment:
I might clarify that this means "commercial support for your version of Python" or something like that. I don't think we're going to get anyone asking for commercial support for Hypothesis on Python 3.3 :-)

Reply:
But if they did, we'd quote an outrageous price rather than saying no?

Reply:
Zac, I am shocked, *shocked* I say, at the very concept!

We would of course quote a *very reasonable* (large) price rather than saying no. ;-)